mmdeploy/core/rewriters/function_rewriter.py

```diff
@@ -1,8 +1,11 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+import types
 from collections import defaultdict
 from typing import (Any, Callable, Dict, List, MutableSequence, Optional,
                     Tuple, Union)
 
+from torch.fx._symbolic_trace import _wrapped_fns_to_patch
```

Comment:
In torch<=1.9, `torch.fx._symbolic_trace` does not exist and  `_wrapped_fns_to_patch` is inside `torch.fx.symbolic_trace`.

Reply:
Nope, update `_wrapped_fns_to_patch` is not necessary if we do not need to trace fx. And we can import `_wrapped_fns_to_patch` from `torch.fx.symbolic_trace` in old torch version.

Reply:
But UT failed
https://app.circleci.com/pipelines/github/open-mmlab/mmdeploy/3442/workflows/1a4e1734-ab69-43c9-a7f6-edb8e8a7cdc2/jobs/9419

Reply:
[This](https://github.com/open-mmlab/mmdeploy/pull/1483/commits/392946a632f83db28008ecb00f6d5d7e8dd3f9d2) change is commited by @pppppM , comment here to discuss the detail.